aspnetcore/fundamentals/index.md

```diff
@@ -4,17 +4,22 @@ author: rick-anderson
 description: Discover the foundational concepts for building ASP.NET Core apps.
 ms.author: riande
 ms.custom: mvc
-ms.date: 08/20/2018
+ms.date: 10/23/2018
 uid: fundamentals/index
 ---
 # ASP.NET Core fundamentals
 
-An ASP.NET Core app is a console app that creates a web server in its `Main` method:
+An ASP.NET Core app is a console app that creates a web server in its `Program.Main` method. The `Main` method is the app's *managed entry point*:
 
 ::: moniker range=">= aspnetcore-2.0"
 
 [!code-csharp[](index/snapshots/2.x/Program.cs)]
 
+The [.NET Core Host](/dotnet/core/tutorials/netcore-hosting), usually *dotnet.exe*:
```

Comment:
`dotnet.exe` is Windows specific name. It is `dotnet` everywhere else. Change this to `usually [dotnet command](/dotnet/core/tools/dotnet)` ?

Reply:
`netcore-hosting` is a very advanced topic. I am not sure whether it is appropriate to link it from here. Maybe just say `NET Core host` without the link? It is what we do elsewhere, like https://docs.microsoft.com/en-us/dotnet/core/deploying/index